internal/sanitize.go

```diff
@@ -24,6 +24,15 @@ const labelKeySizeLimit = 100
 // Sanitize returns a string that is trunacated to 100 characters if it's too
 // long, and replaces non-alphanumeric characters to underscores.
 func Sanitize(s string) string {
+	if len(s) == 0 {
+		return s
+	}
+	if unicode.IsDigit(rune(s[0])) {
+		s = "key_" + s
+	}
+	if s[0] == '_' {
```

Comment:
This misses the case where the string begins with a character that is not an underscore, but will be an underscore after escaping.

Reply:
https://github.com/census-instrumentation/opencensus-go/pull/186 is sent.